L/louvain/build_tarballs.jl

```diff
@@ -14,9 +14,8 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/louvain/
 make -j${nproc}
```

Comment:
Uhm, this is forcing `g++` as compiler everywhere, which is wrong

Reply:
```suggestion
make -j${nproc} CXX="${CXX}"
```

Reply:
Side note: https://github.com/KrainskiL/louvain/blob/9e3e081f054e531e32af38fe71d2991f954ea347/Makefile#L4 `-O5` looks a bit too high

Reply:
https://github.com/KrainskiL/louvain/pull/1